webui/module/Client/src/Client/Model/ClientModel.php

```diff
@@ -120,10 +120,8 @@ public function getClientBackups(&$bsock = null, $client = null, $fileset = null
                         $result = $bsock->send_command($cmd, 2);
                         $fileset = \Zend\Json\Json::decode($result, \Zend\Json\Json::TYPE_ARRAY);
                         $filesets_pluginjob[$backup['fileset']] = false;
-                        if (array_key_exists("plugin", $fileset['result']['filesets'][$backup['fileset']]['include'][0])) {
-                            if (!empty($fileset['result']['filesets'][$backup['fileset']]['include'][0]['plugin'])) {
-                                $filesets_pluginjob[$backup['fileset']] = true;
-                            }
+                        if (!empty($fileset['result']['filesets'][$backup['fileset']]['include'][0]['plugin'])) {
+                            $filesets_pluginjob[$backup['fileset']] = true;
```

Comment:
No. For the check `array_key_exists("plugin", $fileset['result']['filesets'][$backup['fileset']]['include'][0]`, the structure `array($fileset['result']['filesets'][$backup['fileset']]['include'][0])` must exist, otherwise it throws an exception, while `empty` is always true, even if the (sub-)structure does not exists ... what should I say, it's PHP ...